ES/ESdd025.xml

```diff
@@ -225,95 +252,99 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                           <q xml:lang="gez">ወበዛቲ: ዕለት: ቅዳሴ: ቤተ: ክርስቲያኑ: ለአቡነ: አረጋዊ: <gap reason="ellipsis" resp="MK"/></q>
                         </item>
                         <item xml:id="a3">
-                           <locus target="#215rc">f. 215rc</locus>
-                           <desc type="GuestText"><title ref="LIT4630Zaaqrabku">Za-ʾaqrabku māḥleta… “The song I am offering…” </title></desc>
+                           <locus target="#215rc"/>
+                           <desc type="GuestText"><title ref="LIT4630Zaaqrabku"/></desc>
                            <q xml:lang="gez">
-                              ዘአቅረብኩ᎓ ማኅሌተ᎓ አዘኪርየ᎓ አእላፈ᎓ እምእለ᎓
-                              ተጸምዱከ᎓ ዘልፈ። ለለይትነበብ᎓ ተወከፈ᎓ ዘንዴትየ᎓ መጽሐፈ᎓ እንተ᎓ ረሰይከ᎓ እግዚኦ᎓ ጸሪቀ᎓ መበለት᎓
-                              ውኩፈ። እምእለ᎓ አብኡ᎓ ብዑላን᎓ እምብዕሎሙ᎓ ዘተርፈ። ነቢያት᎓ ቅዱሳን᎓ ወሐዋርያት᎓ ንጹሐን᎓ ሰማዕት᎓
-                              ወጻድቃን᎓ ወመላእክት᎓ ትጉሃን᎓ ደናግል᎓ ዓዲ᎓ ወመነኮሳት᎓ ኄራን᎓ ባርኮ᎓ ባርኩ᎓ ጉባኤ᎓ ዛቲ᎓ መካን᎓
-                              እስከ᎓ አረጋዊ᎓ ልኂቅ᎓ ወንዑስ᎓ ሕፃን᎓ ለዘጸሐፎ᎓ በክርታስ᎓ ወለዘአጽሐፎ᎓ እንዘ᎓ ይደርስ። ለዘአንበቦ᎓
-                              ወለዘተርጐሞ᎓ በልሳን᎓ ሐዲስ᎓ ወለዘሰምዓ᎓ ቃላቲሁ᎓ በዕዝነ᎓ መንፈስ᎓ በጸሎተ᎓ እሙ᎓ ማርያም᎓ ዓራቂተ᎓
-                              ኵሉ᎓ እምባዕስ᎓ ኅቡረ᎓ ይምሐረነ᎓ ኢየሱስ᎓ ክርስቶስ።</q>
+                              ዘአቅረብኩ፡ ማኅሌተ፡ አዘኪርየ፡ አእላፈ፡ እምእለ፡
+                              ተጸምዱከ፡ ዘልፈ፡፡ ለለይትነበብ፡ ተወከፈ፡ ዘንዴትየ፡ መጽሐፈ፡ እንተ፡ ረሰይከ፡ እግዚኦ፡ ጸሪቀ፡ መበለት፡
+                              ውኩፈ፡፡ እምእለ፡᎓ አብኡ፡ ብዑላን፡ እምብዕሎሙ፡ ዘተርፈ። ነቢያት፡ ቅዱሳን፡ ወሐዋርያት፡ ንጹሐን፡ ሰማዕት፡
+                              ወጻድቃን፡ ወመላእክት፡ ትጉሃን፡ ደናግል፡ ዓዲ፡ ወመነኮሳት፡ ኄራን፡ ባርኮ፡ ባርኩ፡ ጉባኤ፡ ዛቲ፡ መካን፡
+                              እስከ፡ አረጋዊ፡ ልኂቅ፡ ወንዑስ፡ ሕፃን፡ ለዘጸሐፎ፡ በክርታስ፡ ወለዘአጽሐፎ፡ እንዘ፡ ይደርስ። ለዘአንበቦ፡
+                              ወለዘተርጐሞ፡ በልሳን፡ ሐዲስ፡ ወለዘሰምዓ፡ ቃላቲሁ፡ በዕዝነ፡ መንፈስ፡ በጸሎተ፡ እሙ፡<hi rend="rubric">ማርያም፡</hi> ዓራቂተ፡
+                              ኵሉ፡ እምባዕስ፡ ኅቡረ፡ ይምሐረነ፡ ኢየሱስ፡ ክርስቶስ።
+                           </q>
                         </item>
-                        <item xml:id="e1"><desc>The title ስንክሳር and the internal number of the monastic
-                           library (424) are written on <locus target="#1r">1r</locus>.  </desc></item>
-                        <item xml:id="e2"><desc>Shelfmark C3-IV-259  on <locus target="#3r">3r</locus></desc></item>
-                        <item xml:id="e3"><desc>Corrections written
-                           interlineally: <locus target="#33ra">33ra</locus>, 52rb, 104va,
-                           138rc.</desc></item>
-                        <item xml:id="e4"><desc>Minor notes in pen: <locus target="#211vc">211vc</locus>,
-                           212r.</desc></item>
-                        <item xml:id="e5"><desc> Stamp: <locus target="#214v">214v</locus>, 215r.</desc></item>
+                        
+                        <item xml:id="e1">
+                           <locus target="#1r"/>
+                           <desc type="StampExlibris">The internal number of the monastic library, 424.</desc></item>
```

Comment:
I think I often encoded the actual number that the note consists of as a `<q>`, so in this case:

```
<desc type="StampExlibris">The internal number of the monastic library.</desc>
<q xml:id="en">424</q>
```

Maybe @thea-m knows what is preferable?

Reply:
Yes, this would be more precise. But there is some variation in particular in extras encoding present in many records.